packages/@uppy/dashboard/src/Dashboard.jsx

```diff
@@ -461,23 +463,12 @@ export default class Dashboard extends UIPlugin {
 
   disableAllFocusableElements = (disable) => {
     const focusableNodes = toArray(this.el.querySelectorAll(FOCUSABLE_ELEMENTS))
+      .filter(node => !node.classList.contains('uppy-Dashboard-close'))
     if (disable) {
-      focusableNodes.forEach((node) => {
-        // save previous tabindex in a data-attribute, to restore when enabling
-        const currentTabIndex = node.getAttribute('tabindex')
-        if (currentTabIndex) {
-          node.dataset.inertTabindex = currentTabIndex // eslint-disable-line no-param-reassign
-        }
-        node.setAttribute('tabindex', '-1')
-      })
-    } else {
-      focusableNodes.forEach((node) => {
-        if ('inertTabindex' in node.dataset) {
-          node.setAttribute('tabindex', node.dataset.inertTabindex)
-        } else {
-          node.removeAttribute('tabindex')
-        }
-      })
+      this.#disabledNodes = [...focusableNodes]
+      focusableNodes.forEach((node) => node.setAttribute('disabled', ''))
+    } else if (this.#disabledNodes) {
+      this.#disabledNodes.forEach(node => node.removeAttribute('disabled'))
     }
```

Comment:
```suggestion
    }
    for (const node of focusableNodes) {
      node.disabled = disable
    }
```

Reply:
Any reason for `for` vs `forEach` here? More modern?

Reply:
Ah, you replaced the entire block. The focusableNodes won't be populated the second time, because they now have `disabled` and will be ignored.

Reply:
```suggestion
    }
    for (const node of focusableNodes) {
      node.disabled = disable
    }
    if (disable) {
      this.#disabledNodes = focusableNodes
    } else {
      this.#focusableNodes = null
    }
```